File: src/main/java/com/yong/kakaobot/response/Message.java
package com.yong.kakaobot.response;

import lombok.Data;

@Data
public class Message {
	private String text;

	public Message(String content) {
		switch (content) {
			case "뭐해":
				this.text = "일하고있다ㅜㅜ";
				break;
			case "안녕":
				this.text = "안녕!";
				break;
			default:
				this.text = "Hello Kakao Bot";
		}
	}
}

File: src/main/java/com/yong/kakaobot/response/Message.kt
package com.yong.kakaobot.response

data class Message(private val param: String) {
    val text:String = when(param) {
        "뭐해" -> "일하고있다ㅜㅜ"
        "안녕" -> "안녕!"
        else -> "Hello KaKao Bot!"
    }
}